backend/src/openarchiefbeheer/templates/admin/base_site.html

```diff
@@ -15,8 +15,18 @@
     <link href="{% static 'ico/favicon-32x32.png' %}" sizes="32x32" rel="icon" type="image/png" />
     <link href="{% static 'ico/favicon.png' %}" sizes="16x16" rel="icon" type="image/png" />
     <link href="{% static 'ico/favicon.png' %}" rel="shortcut icon">
-{% endblock %}
 
+    <script>
+      /**
+       * This clears the user (whoAmI) cache and should be triggered when hijacking a user.
+       * NOTE: This only works when the frontend runs on the same host (domain) as the backend.
+       */
```

Comment:
At the moment the project is set up so that it can work with backend/frontend hosted on different domains. I wonder if it would make sense to just refresh the cache of the whoami endpoint everytime the frontend goes to the root path `/`? I think that you automatically get redirected here on hijack :thinking: